pkg/operation/seed/seed.go

```diff
@@ -265,6 +254,25 @@ func BootstrapCluster(seed *Seed, k8sGardenClient kubernetes.Client, secrets map
 	if err != nil {
 		return err
 	}
+	am := map[string]interface{}{}
+	if alertingSMTPKeys := common.GetSecretKeysWithPrefix(common.GardenRoleAlertingSMTP, secrets); len(alertingSMTPKeys) > 0 {
+		emailConfigs := []map[string]interface{}{}
```

Comment:
since you know the final size of the slice, initialize with zero length and a capacity equal to the length of `alertingSMTPKeys`

Reply:
> since you know the final size of the slice, initialize with zero length and a capacity equal to the length of alertingSMTPKeys

This means you should do it like
```go
emailConfigs := make([]map[string]interface{}, 0, len(alertingSMTPKeys))
```